.gitattributes

```diff
@@ -74,6 +74,7 @@
 
 ## Generated SDK files
 packages/**/lib/src/sdk/src/**              linguist-generated
+packages/auth/amplify_auth_cognito_dart/lib/src/sdk/sdk_exception.dart linguist-generated
```

Comment:
Q: why do we need to add this?

Reply:
The file is generated but doesn't have the .g.dart at the end